src/fiatExchanges/quotes/ExternalQuote.ts

```diff
@@ -43,16 +42,20 @@ export default class ExternalQuote extends NormalizedQuote {
   quote: RawProviderQuote | SimplexQuote
   provider: FetchProvidersOutput
   flow: CICOFlow
+  tokenId: string
   constructor({
     quote,
     provider,
     flow,
+    tokenId,
   }: {
     quote: RawProviderQuote | SimplexQuote
     provider: FetchProvidersOutput
     flow: CICOFlow
+    tokenId: string
   }) {
     super()
+    this.tokenId = tokenId
```

Comment:
Should something throw here if tokenId is invalid? If so we should add a test for this.

Reply:
wdym invalid exactly? invalid as in "this token is definitely not supported for quotes"?

also total nit, but can move this assignment down to the bottom of the constructor with the other instance variables.